runtime-builder/go-1.10.yaml

```diff
@@ -1,6 +1,6 @@
 steps:
 - name: 'gcr.io/gcp-runtimes/go1-builder:1.10'
-- name: 'gcr.io/cloud-builders/docker:latest'
-  args: ['build', '-t', '$_OUTPUT_IMAGE', '.']
+- name: 'gcr.io/kaniko-project/executor:v0.4.0'
+  args: ['--destination=$_OUTPUT_IMAGE']
 images:
```

Comment:
None of the yaml files have this images line anymore. I'm guessing we should remove it here as well.